staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -29,6 +29,7 @@ const (
 	DeprecatedRollbackTo           = "deprecated.deployment.rollback.to"
 	DeprecatedTemplateGeneration   = "deprecated.daemonset.template.generation"
 	StatefulSetPodNameLabel        = "statefulset.kubernetes.io/pod-name"
+	StatefulSetPodIndexLabel       = "apps.kubernetes.io/pod-index"
```

Comment:
if the value is not statefulset-specific, the constant name probably shouldn't be either... are things like indexed jobs going to start setting this label as well?

Reply:
not, it is per api-group, job will use `batch.kubernetes.io`; so lets name the constant here `PodIndexLabel` (it is already under the apps pkg)

Reply:
Renamed it to `PodIndexLabel`